services/api-db/docker-entrypoint-initdb.d/01-migrations.sql

```diff
@@ -541,7 +541,7 @@ CREATE OR REPLACE PROCEDURE
         AND column_name = 'scope'
     ) THEN
       ALTER TABLE `env_vars`
-      ADD `scope` ENUM('global', 'build', 'runtime') NOT NULL DEFAULT 'global';
+      ADD `scope` ENUM('global', 'build', 'runtime', 'docker') NOT NULL DEFAULT 'global';
```

Comment:
this unfortunately is not gonna work. As the `if` in this procedure just checks if the `scope` column exist or not, and currently all lagoons will have that column already existing it will never run the `ADD`.

Instead we should create a new procedure which specifically checks if `docker` (or `registry` - see above) exists in the list of enums of column `scope` and if not add it.

Reply:
```suggestion
      ADD `scope` ENUM('global', 'build', 'runtime', 'container_registry') NOT NULL DEFAULT 'global';
```